src/explore-education-statistics-frontend/src/modules/table-tool/components/FixedMultiHeaderDataTable.tsx

```diff
@@ -61,7 +61,7 @@ const FixedMultiHeaderDataTable = forwardRef<HTMLElement, Props>(
             ref={mainTableRef}
           />
         </div>
-        {footnotes && (
+        {footnotes && footnotes.length > 0 && (
```

Comment:
Rather than have to check footnotes twice, we could make the default value an empty array

Reply:
Changed

Reply:
Don't think this has actually been done. You need to actually assign `footnotes = []`